test/controllers/api/v2/hosts_bulk_actions_controller_test.rb

```diff
@@ -124,14 +124,22 @@ def test_install_errata
     end
 
     def test_destroy_hosts
-      assert_async_task(::Actions::BulkAction, ::Actions::Katello::Host::Destroy, [@host1, @host2])
+      assert_async_task(::Actions::BulkAction) do |action_class, hosts|
+        assert_equal action_class, ::Actions::Katello::Host::Destroy
+        assert_equal hosts, [@host1, @host2]
+      end
 
       put :destroy_hosts, :included => {:ids => @host_ids}, :organization_id => @org.id
       assert_response :success
     end
 
     def test_content_view_environment
-      assert_async_task(::Actions::BulkAction, ::Actions::Katello::Host::UpdateContentView, [@host1, @host2], @view_2.id, @library.id)
+      assert_async_task(::Actions::BulkAction) do |action_class, hosts, view_id, library_id|
+        assert_equal action_class, ::Actions::Katello::Host::UpdateContentView
+        assert_equal hosts, [@host1, @host2]
+        assert_equal view_id, @view_2.id
+        assert_equal library_id, @library.id
```

Comment:
Nitpick, assert_equal takes  (expected, actual) as arguments, but you've got them reversed.  Functionally its the same, but output will look wrong such as:

Expected foo to equal bar.